tools/swift-reflection-test/swift-reflection-test.c

```diff
@@ -441,6 +453,8 @@ int reflectExistential(SwiftReflectionContextRef RC,
 int doDumpHeapInstance(const char *BinaryFilename) {
   PipeMemoryReader Pipe = createPipeMemoryReader();
 
+#if defined(_WIN32)
```

Comment:
Right here's where I'd want a comment, I think, even if it's just "Windows doesn't have an equivalent to `fork`, so we'll have to do this in-process."